float64 x
float64 y
float64 z
float64 qw
float64 qx
float64 qy
float64 qz
string planning_method
float64 velocity_scaling
float64 x_tolerance
float64 y_tolerance
float64 z_tolerance
---
float64 rt
